test_local_deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-docker run --rm -v `pwd`:/srv/jekyll -i -t -p 127.0.0.1:4000:4000 jekyll/jekyll:pages
+docker run --rm -v `pwd`:/srv/jekyll -i -t -p 127.0.0.1:4000:4000 jekyll/jekyll:pages jekyll serve --watch
```

Comment:
I would suggest to separate this change since it is unrelated to the time article.

Reply:
Yeah, I'll do that.

Reply:
Rebase since #145 is merged?